bonus.md

```diff
@@ -4,10 +4,22 @@ Edit this file to answer the following questions. Give examples. You may need to
 
 1. Why is Hash.fetch useful?
 
+Hash fetch is useful because it allows us to set default values from our args hash without running into issues with booleans. Using the logical or can cause problems, especially when trying to set to boolean `false` or `nil`. Using `||`, those will never be true and thus, never be assigned. 
+
```

Comment:
Yup.  What happens when you use regular hash syntax and a key is not present? What does fetch do if it's not present (and no default is provided?)